examples/directions-two-in-one.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
```

Comment:
Was this relevant for the branch?
Otherwise it shouldn't be in the refactor branch.
I would move it to another local branch and add after PR.

Reply:
The file was used to manually test if different metadata is created for multiple requests in one file, before cypress testing was implemented.
I have removed it and written a cypress test instead.